packages/content-api/src/ApiContent.format.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { ApiKeyword, ApiContent } from './ApiContent.format';
+import { ThrowReporter } from 'io-ts/es6/ThrowReporter';
+import { map } from 'fp-ts/es6/Either';
+
+type ApiContent = import('./ApiContent.types').ApiContent;
+type ApiKeyword = import('./ApiContent.types').ApiKeyword;
+
+describe('ApiKeywords', () => {
+  it('should have a mandatory label', () => {
+    const apiKeyword = ApiKeyword.decode({
+      label: 'my-keyword',
+    });
+
+    map((keyword: ApiKeyword) => expect(keyword).toMatchSnapshot())(apiKeyword);
```

Comment:
Würde ich entweder mit `it.each` machen oder einfach 

```suggestion
    expect(apiKeyword).toMatchSnapshot();
```

?